editoast/schemas/src/rolling_stock/supported_signaling_system.rs

```diff
@@ -0,0 +1,28 @@
+use serde::Deserialize;
+use serde::Serialize;
+use strum::Display;
+use strum::EnumString;
+use strum::FromRepr;
+use utoipa::ToSchema;
+
+use crate::rolling_stock::EtcsBrakeParams;
+
+#[derive(
+    Clone, Debug, PartialEq, Deserialize, Serialize, Display, EnumString, FromRepr, ToSchema,
+)]
+pub enum SignalingSystem {
+    BAL,
+    BAPR,
+    TVM300,
+    TVM430,
+}
+
+#[derive(Clone, Debug, PartialEq, Deserialize, Serialize, Display, ToSchema)]
+#[serde(untagged)]
+#[allow(clippy::large_enum_variant)]
+pub enum RollingStockSupportedSignalingSystem {
```

Comment:
Since we are in the `rolling_stock` module, no need for the `RollingStock` prefix.


```suggestion
pub enum SupportedSignalingSystem {
```